services/kubernetesmisc/src/handlers/resticRestore.ts

```diff
@@ -86,8 +86,16 @@ async function resticRestore (data: any) {
 
   try {
 
+    let baasBucketName = ""
+    for (let variable of project.projectInfo.envVariables) {
```

Comment:
The result of `getOpenShiftInfoForProject` is an object keyed by `project`. The [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) on line 78 creates a local variable `projectInfo` with the contents of `project` so you'd want to use `projectInfo.envVariables` here.